src/Maker/MakeAuthenticator.php

```diff
@@ -253,6 +255,20 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
         } catch (YamlManipulationFailedException) {
         }
 
+        if ($this->supportsLogoutRouteLoader()) {
+            $yamlData = [];
+            if (file_exists($path = 'config/routes/security.yaml')) {
+                $yamlData = Yaml::parse($this->fileManager->getFileContents($path)) ?? [];
+            }
+            if (!(\array_key_exists('_symfony_logout', $yamlData) && 'security.route_loader.logout' !== $yamlData['_symfony_logout']['ressource'] ?? null)) {
+                $yamlData['_symfony_logout'] = [
+                    'resource' => 'security.route_loader.logout',
+                    'type' => 'service',
+                ];
+            }
+            $generator->dumpFile($path, Yaml::dump($yamlData));
+        }
```

Comment:
To follow on Wouter's comment. I would:

A) Check if `config/routes/security.yaml` exists
B) If it does, do nothing. We will assume they have the latest recipe from https://github.com/symfony/recipes/pull/1244
C) If it does not exist, we print a message:

> Be sure to add the "logout" route to config/routes/security.yaml or upgrade the symfony/security-bundle recipe.

Reply:
I couldn't find a way to check for the presence of the loader, so I ended up using using your suggestion. If users moved the definition elsewhere they should now how to interpret the message.